uber/model_checks.py

```diff
@@ -120,7 +144,7 @@ def emergency_contact(attendee):
 @validation.Attendee
 @ignore_unassigned_and_placeholders
 def cellphone(attendee):
-    if attendee.cellphone and _invalid_phone_number(attendee.cellphone):
+    if attendee.cellphone and _invalid_phone_number(attendee.cellphone) and not c.COLLECT_FULL_ADDRESS:
```

Comment:
Hmm, so does this mean that when we collect the full address, we just don't validate phone numbers?

I don't have a huge problem with that (since it doesn't affect MAGFest), but that's extremely counter-intuitive, and I'd want to leave an ALL-CAPS-WARNING in ``configspec.ini`` indicating that our current implementation stops validating phone numbers if attendee address collection is turned on.